AcadeMeData/tests/test_msgboard_feature.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+from AcadeMeData.forms import MessageForm
+from django.contrib.messages import get_messages
+
+
+@pytest.mark.django_db
+def test_msgboard_login(client, user_example):
+    client.force_login(user_example.user)
+    response = client.get("/msgboard/")
+    assert response.status_code == 200
+
+
+@pytest.mark.django_db
+def test_msgboard_form_instance(client, user_example):
+    client.force_login(user_example.user)
+    response = client.get("/msgboard/")
+    form = response.context["form"]
+    assert isinstance(form, MessageForm)
+
+
+@pytest.mark.django_db
+def test_msgboard_empty_message_when_page_loads(client, user_example):
+    client.force_login(user_example.user)
+    response = client.get("/msgboard/")
+    messages = list(get_messages(response.wsgi_request))
```

Comment:
Consider this 
 len(response.context['messages'])  instead

Reply:
The post test is giving me a hard time, checking everything from a to z, it only fills the form on /msgboard/ but not posting it and adding to the db
I think this is something that we can deal with later on, everything works manually, posting messages to msgboard is working perfectly 
(messages are being added to the DB) and other tests are fine too.
In order to get ahead with the project I would love for this pr to be merged and promise to deal with this specific test later on.

Reply:
Tests are the most important part of the code, more than the functionality itself; it is not something we usually push aside for later. 
your code may work now, but a future feature (even if not related) may break it without you even noticing.

Reply:
I understand and agree, but Its taking me to much time to solve it,
I have read the docs, countless stackoverflow questions
I have consulted mentors, friends, different teams and no one could help 🥲 
If you want I can show you what the problem is via zoom

Reply:
I really do get your frustration. but it does not meet our definition of done. 
sometimes it is better not having a feature than having something that does not meet the standards we decided on (good coverage of tests in this situation). 
I am glad to hear it made you consult others, as this is a good way to learn. 

I would've liked to see the problem, but I don't think I'll be available for this before the demo...

Reply:
I'll try again 🥲 
thx